ui/src/playlist/PlaylistShow.js

```diff
@@ -4,6 +4,7 @@ import {
   ShowContextProvider,
   useShowContext,
   useShowController,
+  Pagination as RaPagination,
```

Comment:
To be consistent with the other views, you should use our own `ui/src/common/Pagination.js`, and not the default RA Pagination

Reply:
I've updated the pagination component.